index.html

```diff
@@ -34,58 +40,58 @@ <h1 class="section--title">Beautiful Free Nova template</h1>
           src="./images/screen.png"
           alt="screen with features"
         />
-      </section>
+      </article>
     </header>
     <!-- End of header section -->
     <!-- Main section -->
     <main>
       <!-- Features section -->
       <section class="features__section">
         <h1 class="section--heading">Features</h1>
-        <div class="section--items">
+        <div class="section__items">
```

Comment:
Skoro jest mamy Element o nazwie `section__items` to powinien być w strukturze HTML również `section` - jego nie widzę. 

PS. Prawdopodobnie klasa `section` powinna być w znaczniku `<section>`